### PR TITLE
Capture logger's output in test (issue #58)

### PR DESCRIPTION
**Pull request by david942j**
This change is [Reviewable](https://reviewable.io/reviews/peter50216/pwntools-ruby/84)

### CODE REVIEW

File: test/test_helper.rb [hunk 1]
```diff
@@ -19,5 +19,25 @@ def before_setup
...
+    def log_stdout(&block)
+      log_hook($stdout, &block)
+    end
+
+    def log_hook(obj)
+      old = ::Pwnlib::Logger.log.instance_variable_get(:@logdev)
+      ::Pwnlib::Logger.log.instance_variable_set(:@logdev, obj)
+      res = yield
+      ::Pwnlib::Logger.log.instance_variable_set(:@logdev, old)
+      res
```
Comment (peter50216) [inline test/test_helper.rb]:
```ruby
begin
  yield
ensure
  ::Pwnlib::Logger.log.instance_variable_set(:@logdev, old)
end

File: test/test_helper.rb [hunk 2]
```diff
@@ -19,5 +19,25 @@ def before_setup
       # Default to disable coloring for easier testing.
       Rainbow.enabled = false
     end
+
+    # Some methods for hooking logger,
+    # require 'pwnlib/logger' before using these methods.
+
+    def log_null(&block)
+      # Windows not support File.open('/dev/null')
```
Comment (peter50216) [inline test/test_helper.rb]: does not support

File: test/test_helper.rb [hunk 3]
```diff
@@ -19,5 +19,27 @@ def before_setup
       # Default to disable coloring for easier testing.
       Rainbow.enabled = false
     end
+
+    # Some methods for hooking logger,
+    # require 'pwnlib/logger' before using these methods.
+
+    def log_null(&block)
+      # Windows does not support File.open('/dev/null')
```
Comment (peter50216) [inline test/test_helper.rb]: Does File.open(File::NULL) works?

**Comment by hanhanW** [comment]
:lgtm:

**Pull request merged**